test-suite/Makefile

```diff
@@ -242,6 +246,45 @@ $(addsuffix .log,$(wildcard bugs/closed/*.v)): %.v.log: %.v
 	  fi; \
 	} > "$@"
 
+#######################################################################
+# Unit tests
+#######################################################################
+
+UNITTEST_OCAMLOPT=ocamlfind opt
+UNITTEST_CAMLFLAGS=-rectypes
```

Comment:
These flags should also be consistent with the ones used for coq
```
$ cat config/Makefile | grep recty
CAMLFLAGS=-thread -rectypes -w +a-4-9-27-41-42-44-45-48-50 -annot -bin-annot -safe-string -warn-error +a-56
```

Reply:
Also this (lines 11-12 above) will no longer be true because of the unit tests.  (Do we care?)  I can remove this statement.  (It appears I can't add review comments to the PR, perhaps because I'm the submitter.)

```
# This is a standalone Makefile to run the test-suite. It can be used
# outside of the Coq source tree (if BIN is overridden).
```

Reply:
I had in mind to include `configure/Makefile`.
One could also use the output of `coqtop -config` (to be run out of tree, assuming coqtop is in the path and that META.coq is in OCAMLPATH).

I'm not sure the use case is very relevant.

What is dangerous is to compile the unit tests with the wrong ocamlfind (wrong compiler).